src/components/Wowerlay.tsx

```diff
@@ -111,6 +111,26 @@ export const Wowerlay = defineComponent({
     };
   },
   render() {
+    let willBeRendered: null | JSX.Element = null;
+
+    const Renderer = !this.isVisible ? null : (
+      <WowerlayRenderer {...this.$props} {...this.$attrs} onClick={this.handleWowerlayClick}>
+        {this.$slots.default?.()}
+      </WowerlayRenderer>
+    );
+
+    if (this.$props.transition === false) {
+      willBeRendered = Renderer;
+    } else if (typeof this.$props.transition === 'string') {
+      willBeRendered = <Transition name={this.$props.transition}>{Renderer}</Transition>;
+    } else {
+      willBeRendered = (
+        <Transition enterActiveClass={cWowerlayAnimEnter} leaveActiveClass={cWowerlayAnimLeave}>
+          {Renderer}
+        </Transition>
+      );
+    }
```

Comment:
`h`'leri de setup içine alırsak ekstra `this` lere gerek kalmaz.

Reply:
Please english, @necatikcl I think you don't know how render works, render basically works with every update not setup phase. Setup runs only once.

Reply:
And you got a point, Why did I use `this.$props.transition` :D I can use `this.transition`.